docs/sections/user_guide/cli/drivers/ungrib.rst

```diff
@@ -0,0 +1,73 @@
+``ungrib``
+==========
+
+The ``uw`` mode for configuring and running the WRF preprocessing component ``ungrib``.
+
+.. code-block:: text
+
+   $ uw ungrib run --help
```

Comment:
I think the other drivers' docs start with a top-level `--help` example, which here would be `uw ungrib --help`, without naming a specific task, and then give a usage example for the `run` task. Unless there's a reason to depart from convention, maybe we should do so here, too -- IMO it's nice to see the available tasks listed in the docs.

Reply:
Oh, that was an oversight on my part, thanks for the catch!